lib/esp32.toit

```diff
@@ -135,3 +135,74 @@ There is only one RTC memory on the device, so all tasks or processes have
 rtc_user_bytes -> ByteArray:
   #primitive.esp32.rtc_user_bytes
 
+/**
```

Comment:
I think we should move towards encoding the bytes (ubjson) in the VM, so this becomes more similar to profiles, histograms and stack traces.